CONTRIBUTING.md

```diff
@@ -158,23 +152,49 @@ Building and testing are documented in our
 
 ### Style Guides
 
+Style consistency is largely enforced by linters in CI. This section reviews
+some of the rules and gives tips on how to auto-fix most linting errors.
+
 #### Git Commit Messages
 
 A quick summary:
 
 - Separate subject from body with a blank line.
-- Limit the subject line to 80 characters.
-- Prefix the subject line with one of:
-  - "breaks:" if commit implements a non-backward compatible breaking change
-  - "fix:" if commit implements a bugfix
-  - "feat:" if commit implements a new feature
-  - "deps:" if commit updates a dependency
-  - "other:" if commit doesn't fall in any of the above categories
```

Comment:
We never did this. Also, it now vaguely applies to changelog fragments, not commit msgs.